<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Snapshot</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>361</width>
     <height>191</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Export Options</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Height (pixels)</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="heightSpinBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>64</y>
       <width>62</width>
       <height>26</height>
      </rect>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>50000.000000000000000</double>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Keep aspect ratio</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="widthSpinBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>62</width>
       <height>26</height>
      </rect>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>50000.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Width (pixels)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="filenameInput">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>110</y>
       <width>90</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Change file</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="transparentBackgroundCheckBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Transparent Background</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Rendering Options</string>
    </attribute>
    <widget class="QSpinBox" name="sphereSubdivisionsSpinBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>42</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="cylinderSubdivisionsSpinBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>42</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Sphere Subdivisions</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cylinder Subdivisions</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>widthSpinBox</tabstop>
  <tabstop>heightSpinBox</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>filenameInput</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>sphereSubdivisionsSpinBox</tabstop>
  <tabstop>cylinderSubdivisionsSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
